solutions/common-shared-flows/correlation-v1/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "correlation-v1",
+  "version": "1.0.0",
+  "scripts": {
+    "deploy": "npm i && npx apigeetool deploySharedflow -u $APIGEE_USER -p $APIGEE_PASS -o $APIGEE_ORG -e $APIGEE_ENV -n $npm_package_name -V",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "LaughingBiscuit",
+  "license": "NONE",
```

Comment:
I feel like we should specify the appropriate license here given this is published publicly. Or will any parent license declarations at the repo level suffice?